extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -382,6 +402,85 @@ private function disableDiscussion( $cityId ) {
 		}
 	}
 
+	// Gets the most recent post from the specified site and returns whether it was made less than 60 days ago
+	private function isActiveSite($siteId)
+	{
+		$response = $this->getMostRecentPostForSite($siteId);
+
+		if ($response['postCount'] == 0) {
+			// no posts
+			return false;
+		}
+
+		// extract post creation date from response
+		$mostRecentPostCreationDate = $response['_embedded']->{'doc:posts'}[0]->{'creationDate'}->{'epochSecond'};
+
+		$sixtyDaysAgo = time() - 60*60*24*60;
+
+		return $mostRecentPostCreationDate > $sixtyDaysAgo;
+	}
+
+	private function getMostRecentPostForSite($siteId) {
+		$apiClient = $this->getSitesApi()->getApiClient();
+
+		$resourcePathTemplate = "/{siteId}/posts";
+		$httpBody = '';
+
+		// header params
+		$headerParams = ['Content-Type' => $apiClient->selectHeaderContentType(array('application/json'))];
+
+		$headerAccept = $apiClient->selectHeaderAccept(array('application/hal+json'));
+		if (!is_null($headerAccept)) {
+			$headerParams['Accept'] = $headerAccept;
+		}
+
+		// path params
+		$resourcePath = str_replace(
+				"{siteId}",
+				$apiClient->getSerializer()->toPathValue($siteId),
+				$resourcePathTemplate
+		);
+
+		// only need most recent post
+		$queryParams = [
+				'limit' => 1
+		];
+
+		// make the API Call
+		try {
+			list($rawResponse, $statusCode, $httpHeader) = $apiClient->callApi(
+					$resourcePath,
+					'GET',
+					$queryParams,
+					$httpBody,
```

Comment:
It looks like from line 427 the value for `$httpBody` passed is meant to be an empty default however, the `callApi` method reports the parameter as an `array` so the IDE is mildly upset that its getting a string.  Consider setting this value closer to the invocation so that its clear that it doesn't do anything, and what default is being passed, e.g.:
```php
list($rawResponse, $statusCode, $httpHeader) = $apiClient->callApi(
		$resourcePath,
		'GET',
		$queryParams,
		$httpBody = [],
		$headerParams,
		'object'
);
```